package/MDAnalysis/core/universe.py

```diff
@@ -74,6 +74,8 @@
 .. autofunction:: Merge
 
 """
+from cStringIO import InputType
```

Comment:
This is going to break in python3 where `StringIO` and its relatives moved to the `io` module, and where the base class is `io.IOBase`.

Something like follow would hopefully work:

```python
try:
    from cStringIO import InputType as _BaseForIO
except ImportError:
    from io import IOBase as _BaseForIO

isinstance(args[0], _BaseForIO)
```